acceptance/allocator_terraform/load.tf

```diff
@@ -4,6 +4,10 @@ variable "example_block_writer_instances" {
   default = 1
 }
 
+variable "load_duration" {
+  default = "0" # unlimited
```

Comment:
Yeah, it *should* be possible to always print the stats when block writer exists (short of being `SIGKILL`ed). I think that would be preferable, especially if it avoids the double-timeout handling and awkward terrafrorm plumbing here.